README.md

```diff
@@ -64,7 +64,13 @@ jwt({ secret: Buffer.from('shhhhhhared-secret', 'base64'),
       algorithms: ['RS256'] })
 ```
 
-Optionally you can make some paths unprotected as follows:
+To only protect specific paths (e.g. beginning with `/api`), use [express router](https://expressjs.com/en/4x/api.html#app.use) like so:
+
+```javascript
+app.use('/api', jwt({ secret: 'shhhhhhared-secret', algorithms: ['HS256']}));
+```
+
+Or, the other way around, if you want to to make some paths unprotected as follows:
```

Comment:
`Or, **optionally**, if you want **to make** some paths unprotected **proceed** as follows:`

- Remove duplicated `to`
- Change `the other way around` with initial `optionally` keyword
- Add `proceed` verb to make a better sentence unless I'm mistaken

Reply:
What's a "keyword"?  And what does "optionally" add to the "if you want" part? 🤔  Here "the other way around" refers to the opposites of opting in or opting out of requiring auth.